cypress/support/e2e.ts

```diff
@@ -15,6 +15,10 @@
 
 import './commands';
 
+before(() => {
+	cy.resetAll();
```

Comment:
Perhaps we should remove this `resetAll` command completely since it's only used here now, and it's prone to cause issues if used elsewhere

Reply:
I wanted to remove it, but it's unfortunately still used in the smoke test.
Maybe we should discuss removing the smoke test, since we run the full suite on every PR.